pkg/network/metrics/metrics.go

```diff
@@ -0,0 +1,58 @@
+package metrics
+
+import (
+	"context"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+)
+
+// Service serves metrics provided by prometheus.
+type Service struct {
+	*http.Server
+	config PrometheusConfig
+}
+
+// PrometheusConfig config for Prometheus used for monitoring.
+// Additional information about Prometheus could be found here: https://prometheus.io/docs/guides/go-application.
+type PrometheusConfig struct {
+	Enabled bool   `yaml:"Enabled"`
+	Port    string `yaml:"Port"`
+}
+
+// NewMetricsService created new service for gathering metrics.
+func NewMetricsService(cfg PrometheusConfig) *Service {
+	return &Service{
+		&http.Server{
+			Addr:    ":" + cfg.Port,
+			Handler: promhttp.Handler(),
+		}, cfg,
+	}
+}
+
+// Start runs http service with exposed /metrics endpoint on provided in configuration port.
+func (ms *Service) Start() {
+	if ms.config.Enabled {
+		err := ms.ListenAndServe()
+		if err != nil {
+			log.WithFields(log.Fields{
+				"endpoint": ms.Addr,
+			}).Info("metrics service up and running")
+		} else {
+			log.Warn("metrics service couldn't start on provided port")
+		}
+	} else {
+		log.Infof("metrics service hasn't started since it's disabled in configuration")
```

Comment:
I'm not sure it's worth `Info` level.

Reply:
so you prefer not to know about it at all? Or what's your suggestion?

Reply:
Well, if it's configured this way it's configured this way. On the other hand, it's not hurting anyone being printed, maybe just leave it as is.